### PR TITLE
change secret name to align with storage class name

### PR DESCRIPTION
**Pull request by khushboo-rancher**
Update the storage class parameter.

And I thought same name of secret aligned with the above storage class example will make more sense. :) 

Signed-off-by: Khushboo <fnu.khushboo@suse.com>

### CODE REVIEW

File: enhancements/2021.07.26-pv-encryption.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ This type of setup means that all volumes share the same encryption key.
 apiVersion: v1
 kind: Secret
 metadata:
-  name: longhorn-crypto
```
Comment (innobead) [inline enhancements/2021.07.26-pv-encryption.md:83]: Let's keep this as the same as examples.

**Review by innobead — CHANGES_REQUESTED** [review]

File: enhancements/2021.07.26-pv-encryption.md (hunk 1 above)
Comment (khushboo-rancher) [reply enhancements/2021.07.26-pv-encryption.md:83]: Done

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@khushboo-rancher the branch has conflicts, needs to resolve first before merging.

**Review by joshimoo — DISMISSED** [review]
LGTM

**Review by innobead — APPROVED** [review]

**Pull request merged**